development/tools/jobs/kyma/kyma_integration_test.go

```diff
@@ -242,7 +242,7 @@ func TestKymaIntegrationJobPeriodics(t *testing.T) {
 	tester.AssertThatHasExtraRefs(t, disksCleanerPeriodic.JobBase.UtilityConfig, []string{"test-infra", "kyma"})
 	assert.Equal(t, "eu.gcr.io/kyma-project/prow/buildpack-golang:0.0.1", disksCleanerPeriodic.Spec.Containers[0].Image)
 	assert.Equal(t, []string{"bash"}, disksCleanerPeriodic.Spec.Containers[0].Command)
-	assert.Equal(t, []string{"-c", "development/disks-cleanup.sh -project=${CLOUDSDK_CORE_PROJECT} -dryRun=false -diskNameRegex='^gke-(gkeint|gke-upgrade).*[-]pvc[-]'"}, disksCleanerPeriodic.Spec.Containers[0].Args)
+	assert.Equal(t, []string{"-c", "development/disks-cleanup.sh -project=${CLOUDSDK_CORE_PROJECT} -dryRun=false -diskNameRegex='^gke-gkeint|gke-upgrade|weekly|nightly|gke-central' -ageInHours=100"}, disksCleanerPeriodic.Spec.Containers[0].Args)
```

Comment:
While this solves a problem every ~5 days now, we could run into creating a lot of disks in the short term. And in the meantime ramp up costs on disks that should only be around for ~3 hours. These disks would now, if not deleted by the resource creating it dangle for ~5 days before getting deleted. Maybe we handle weekly different than the rest.

Reply:
are you suggesting we lower this to two hours?